module_properties/utils/properties.py

```diff
@@ -0,0 +1,253 @@
+import json
+import requests
+from rdkit import Chem
+from rdkit.Chem.Descriptors import (
+    ExactMolWt,
+    HeavyAtomMolWt,
+    MolWt,
+    MaxPartialCharge,
+    MinPartialCharge,
+    NumRadicalElectrons,
+    NumValenceElectrons,
+)
+from rdkit.Chem.Crippen import MolLogP
+from rdkit.Chem.QED import default, weights_max, weights_none
+from rdkit.Chem.Lipinski import (
+    NHOHCount,
+    NOCount,
+    NumHAcceptors,
+    NumHDonors,
+    NumHeteroatoms,
+    NumRotatableBonds,
+    RingCount,
+)
+from module_properties.utils.errors import (
+    InvalidJsonError,
+    StructNotFoundError,
+    InChiKeyNotFoundError,
+    InvalidPubChemPropertyError,
+    InvalidRequestError,
+    InvalidSMILESError,
+)
+from module_identifiers.docs.docs import Compounds
+from module_properties.docs.docs import Properties, Pubchem, RDKit
+
+
+def __format_request_url(inchikey, selected_properties):
```

Comment:
why dunder names?

Reply:
Its a helper method of the module so __ to signify its supposed to be private. That's what I have always used for private methods. If we need to decide on a different signification for private methods we can do so

Reply:
name mangling is a hack to create private methods- even then it doesn't actually enforce privacy. If you want privacy, use a different language. The best you can do is use a sunder name to *suggest* privacy